### PR TITLE
Add a basic QueryBuilder unit test

### PR DESCRIPTION
**Pull request by ranquild**
Adds a skeleton of a test suite to allow adding other tests easier.

How to verify:
- The new test should pass

- - -
This change is [Reviewable](https://reviewable.io/reviews/metabase/metabase/30481)

### CODE REVIEW

**Review by oisincoveney — APPROVED** [review]
Tested locally, works perfectly 😄

File: frontend/src/metabase-types/api/alert.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export type AlertId = number;
+
+export interface Alert {
```
Comment (uladzimirdev) [inline frontend/src/metabase-types/api/alert.ts:3]: we have already Alert type in scope of my branch

Comment (ranquild) [reply frontend/src/metabase-types/api/alert.ts:3]: @uladzimirdev would that be ok if we merge this branch first to unblock @oisincoveney and you could add other alert fields in your PR?

Comment (uladzimirdev) [reply frontend/src/metabase-types/api/alert.ts:3]: I'm waiting for one test job to pass, so most likely my PR will be a bit faster to merge

Comment (uladzimirdev) [reply frontend/src/metabase-types/api/alert.ts]: merged master into your branch

Comment (ranquild) [reply frontend/src/metabase-types/api/alert.ts]: Thanks!

**Pull request merged**